client/src/stores/history/historyItemsStore.js

```diff
@@ -28,7 +28,9 @@ export const useHistoryItemsStore = defineStore("historyItemsStore", {
         getHistoryItems: (state) => {
             return (historyId, filterText) => {
                 const itemArray = state.items[historyId] || [];
-                const filters = HistoryFilters.getFilters(filterText).filter((filter) => !filter.includes("related"));
+                const filters = HistoryFilters.getFilters(filterText).filter(
+                    (filter) => !filter[0].includes("related")
```

Comment:
I'm happy to merge this with manual verification for the release, but it might be really nice to have store specific tests applying the filtering here if you had time to take a pass at that.  I know we have tests for HistoryFilters, so maybe pushing this filter specific logic into there and testing it there makes sense, too?